### ISSUE TITLE
Feature request: Set close-on-idle time

### ISSUE THREAD

**Issue by BioTurboNick** [fonsp/Pluto.jl#2236]
When using Pluto on a shared server, it's common for people to leave notebooks running after they don't need them.

It would be nice to be able to automatically close notebooks that have been idle for a set time. Perhaps through an argument passed to the `run` method.

**Comment by Pangoraw** [comment]
@pankgeorg is this something that can currently be built with Pluto events ?

**Comment by pankgeorg** [comment]
Yes of course! There are many events that can be used (like save or run) but what does idle mean? No bond interaction or no web view? The latter would be harder (but we could implement a new event, like OnZeroConnectedClients or something)

**Comment by BioTurboNick** [comment]
No bond interaction/no cells executed would be reasonable for my needs.
